### DISCUSSION TITLE
How can I switch from markedjs to mdx ?

### DISCUSSION THREAD

**Discussion by beumsk** [mdx-js/mdx#2167] [Q&A] [answered]
Hello,
Currently I'm working with markedjs to compile my .md files into html.
It works like this: 
```js
import { marked } from 'marked';
const html = marked.parse(importedMarkdownFileFromNode);
```
How should I achieve the same with mdx? I could not find the way to use mdx in a similar way.
Thanks !

**Comment by wooorm** [comment, marked as answer]
They are very different. Like switching from React to web components or Vue or so.
But, read the docs: first three chapters here should give you everything you need: https://mdxjs.com/docs/.
